docs/installation.md

```diff
@@ -218,19 +218,234 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
 
 ------
 
-### Anaconda
+## Conda (Miniconda or Anaconda)
 
-Freqtrade can also be installed using Anaconda (or Miniconda).
+Freqtrade can also be installed with Miniconda or Anaconda.
+Conda is a package manager and virtual environment manager in one.
+The purpose of installing Conda (Miniconda or Anaconda) is to automatically prepare and manage the extensive library dependencies of the Freqtrade program.
 
-!!! Note
-    This requires the [ta-lib](#1-install-ta-lib) C-library to be installed first. See below.
+You can start with any clean distribution of Debian-based Linux distributions, which includes Ubuntu Linux.
+For other popular Linux distributions, see: https://distrowatch.com/ or https://www.ubuntupit.com/best-debian-based-linux-distributions/
+
+Installation requirement: Your Linux is a Debian based distribution with bash terminal
+
+To get know more about Conda check : https://linuxnetmag.com/miniconda-vs-anaconda/
+
+Shortly : Conda < Miniconda < Anaconda
+
+https://www.dunderdata.com/blog/anaconda-is-bloated-set-up-a-lean-robust-data-science-environment-with-miniconda-and-conda-forge
+
+The difference lies in the number of packages that need to be installed upfront: how heavy/light the installation is. Difference is like ~1:8. It is recommended to install Miniconda instead of Anaconda.
+
+#### 1. Instal Conda
+
+##### Download conda file:
+
+installation file of chosen distribution you can find here
+
+https://docs.conda.io/en/latest/miniconda.html
+
+##### Verify file integrity with SHA256 checksum:
+
+official SHA256 checksum can be found here
+
+https://docs.conda.io/en/latest/miniconda_hashes.html
+
+Navigate to the directory with your file, and run SHA256 checksum of your file, check with following command
+
+```bash
+sha256sum ./Miniconda3-number-of-the-package
+```
+
+both checksum suppose to match
+
+##### Install conda:
+
+While in directory with your file, install conda
+
+```bash
+bash ./Miniconda3-number-of-the-package
+```
+
+Confirm with yes all questions
+
+After installation, it is mandatory to turn your terminal OFF and ON again.
+
+CTRL + D (close terminal), CTRL + ALT + T (open terminal)
```

Comment:
Please link to the Anaconda / miniconta installation documentation instead of writing it down here.

A helper in "pick miniconda instead of conda" is ok - but there's no point in maintaining a seperate documentation for this - as it'll become outdated eventually, and it provides no benefit to the user.

Also, it makes this documentation WAY too long - if as a user, i see an install-instruction of this length, i'll immediately decide to use a different method, just based on the length of the instruction.

Reply:
Conda head `### What is conda` is changed. 
`#### 1. Instal Conda` is changed as well, according to your style, is much shorter now